tests/integration/features/bootstrap/FeatureContext.php

```diff
@@ -48,6 +48,16 @@ public function setOpenedEmailStorage(OpenedEmailStorage $storage): void {
 		$this->openedEmailStorage = $storage;
 	}
 
+	public function sendRequest(string $verb, string $url, $body = null, array $headers = [], array $options = []): void {
+		if (!is_null($this->currentUser)) {
+			$options = array_merge(
+				['cookies' => $this->getUserCookieJar($this->currentUser)],
+				$options
+			);
+		}
+		parent::sendRequest($verb, $url, $body, $headers, $options);
+	}
```

Comment:
Cannot find the usage of this method, can u appoint it, please ?

Reply:
When we need to use a guest account, is the account that don't have an username, we need to send the cookie as the browser do.

The method stay at parent class and by default only set the option `cookie` when exists an username.

The default value of `currentUser` is null.

We have a step to set the user to "":
```gherkin
And as user ""
```

At this point we will need to send the session cookie to backend. If we don't do this, the logic that get the start time by session, don't will work.

Reply:
Maybe we can create a technical debt issue to identify a best way to define this logic.

To me was a bit strange to use the currentUser by this way. It is not clear to anyone who looks this method because why this was made.